draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -2243,7 +2243,7 @@ The Verifier would decode the HS-Disclosures JWT and SD-JWT as follows:
    * explain why JSON-encoding of values is needed
    * explain merging algorithm ("processing model")
    * generalized hash alg to digest derivation alg which also enables HMAC to calculate digests
-   * `sd_digest_derivation_alg` renamed to `sd_digest_derivation_alg`
+   * `sd_digests` renamed to `sd_digest_derivation_alg`
```

Comment:
```suggestion
   * `sd_hash_alg` renamed to `sd_digest_derivation_alg`
```

Reply:
Can't believe I got the name wrong there... 🤦